src/background.c

```diff
@@ -379,6 +382,8 @@ update_background_monitor_properties (void)
       g_variant_builder_init (&app_builder, G_VARIANT_TYPE_VARDICT);
       g_variant_builder_add (&app_builder, "{sv}", "app_id", g_variant_new_string (app_id));
       g_variant_builder_add (&app_builder, "{sv}", "instance", g_variant_new_string (id));
+      if (data->status_message)
+        g_variant_builder_add (&app_builder, "{sv}", "message", g_variant_new_string (data->status_message));
```

Comment:
i guess the "message" key should also be mentioned in the xml interface description

Reply:
It's described in `org.freedesktop.background.Monitor` already